.github/workflows/pr-title-checker.yml

```diff
@@ -14,9 +14,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check PR title validity
-        if: "!contains(github.event.pull_request.title, '[FEATURE]') && !contains(github.event.pull_request.title, '[BUGFIX]') && !contains(github.event.pull_request.title, '[DOCS]') && !contains(github.event.pull_request.title, '[MAINTENANCE]') && !contains(github.event.pull_request.title, '[CONTRIB]') && !contains(github.event.pull_request.title, '[RELEASE]')"
+        if: >-
```

Comment:
This line was getting long so I reformatted. I have verified that the title checker still runs and works.